demo/demo.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'example-app',
+  template: `
+    <div class="examples">
+      <example-a></example-a>
+    
+      <example-b></example-b>
+    
+      <another-example></another-example>
+    
+      <such-example></such-example>
+    
+      <very-example></very-example>
+    
+      <much-example></much-example>
+    
+      <wow-example></wow-example>
```

Comment:
:)

On Wed, Sep 7, 2016, 17:04 Bojan Kogoj <notifications@github.com> wrote:

> In demo/demo.component.ts
> <https://github.com/valor-software/ng2-dragula/pull/396#discussion_r77827799>
> :
>
> > +  selector: 'example-app',
> > +  template: `
> > +    <div class="examples">
> > +      <example-a></example-a>
> > +
> > +      <example-b></example-b>
> > +
> > +      <another-example></another-example>
> > +
> > +      <such-example></such-example>
> > +
> > +      <very-example></very-example>
> > +
> > +      <much-example></much-example>
> > +
> > +      <wow-example></wow-example>
>
> Look at the example names
> <http://sd.keepcalm-o-matic.co.uk/i/such-doge-much-wow.png>
>
> Not saying it should be changed, just funny
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/valor-software/ng2-dragula/pull/396/files/addaa7766570ecc8e21e80447c3146fd7e8fbaf4..aa75c0e367d68322691f1f7843c6aee5b69f05b3#r77827799>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABDk45dmjT-rnp1XV3XsR973e75RWYLWks5qnsRQgaJpZM4J2GHD>
> .
>